browser/ui/webui/settings/brave_settings_localized_strings_provider.cc

```diff
@@ -97,832 +97,809 @@ constexpr char16_t kLeoPrivacyPolicyURL[] =
 void BraveAddCommonStrings(content::WebUIDataSource* html_source,
                            Profile* profile) {
   webui::LocalizedString localized_strings[] = {
-      {"importExtensions", IDS_SETTINGS_IMPORT_EXTENSIONS_CHECKBOX},
-      {"importPayments", IDS_SETTINGS_IMPORT_PAYMENTS_CHECKBOX},
-      {"siteSettingsAutoplay", IDS_SETTINGS_SITE_SETTINGS_AUTOPLAY},
-      {"siteSettingsCategoryAutoplay", IDS_SETTINGS_SITE_SETTINGS_AUTOPLAY},
-      {"siteSettingsAutoplayAllow", IDS_SETTINGS_SITE_SETTINGS_AUTOPLAY_ALLOW},
-      {"siteSettingsAutoplayBlock", IDS_SETTINGS_SITE_SETTINGS_AUTOPLAY_BLOCK},
+    {"importExtensions", IDS_SETTINGS_IMPORT_EXTENSIONS_CHECKBOX},
```

Comment:
Lots of unrelated changes in this file, probably due to IDE auto formatting, please remove these and only change what's necessary here.

Reply:
@yrliou I believe the linting rules changed in a recent Chromium update. The formatting changes were performed via the presubmit checks.

Reply:
Ah okay, just a suggestion for future references, in this case it would be nice to separate lint/format changes to a separate commit to help reviewer seeing what's actually changing here.